src/logging.c

```diff
@@ -44,8 +44,6 @@ int ugly_log(int level, const char *tag, const char *format, ...) {
     case UFATAL:
```

Comment:
We should also remove the UFATAL loglevel completely. And also the https://github.com/texane/stlink/blob/master/include/stlink/logging.h#L36-L37 which are completely unused and dangerous for a lib.